src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_utils.py

```diff
@@ -37,3 +39,87 @@ def create_self_signed_certificate(device_id, valid_days, cert_output_dir):
         'privateKey': key_dump,
         'thumbprint': thumbprint
     }
+
+
+def open_certificate(certificate_path):
+    certificate = ""
+    if certificate_path.endswith('.pem') or certificate_path.endswith('.cer'):
+        with open(certificate_path, "rb") as cert_file:
+            certificate = cert_file.read()
+            try:
+                certificate = certificate.decode("utf-8")
+            except UnicodeError:
+                certificate = base64.b64encode(certificate).decode("utf-8")
+    return certificate
+
+
+def _create_test_cert(cert_file, key_file, subject, valid_days, serial_number):
```

Comment:
What is this used for?  If only for tests, consider moving it into your test directory so that it doesn't find its way into a command some day.

Reply:
Yes, this specific function is only used in tests, I'll separate these out.